### DISCUSSION TITLE
Refresh Token

### DISCUSSION THREAD

**Discussion by ksnarmada** [keycloak/keycloak#33372] [Q&A]
Even after configuring Client Session Max, Client Session Idle, SSO Session Idle and SSO Session Max, from the token endpoint, I always see refresh_expires_in as 0.

**Comment by codespearhead** [comment]
You're probably using an offline_access token, in which case the relevant field is `Offline Session Max Limited`.

You can learn more about it here [1].

[1] https://www.janua.fr/examples-of-offline-token-usage-in-keycloak/
